src/benchmark_mediation.py

```diff
@@ -1036,13 +1064,24 @@ def med_dml(
     var_name += ["mu_t1_m_x", "mu_t0_m_x", "w_t0_x", "w_t1_x", "mu_t1_x", "mu_t0_x"]
     nobs = 0
 
+    # define regularization parameters
     if regularization:
         alphas = ALPHAS
         cs = ALPHAS
     else:
         alphas = [0.0]
         cs = [np.inf]
 
+    # choose forest or regression model
+    if forest:
+        BinaryModel = RandomForestClassifier(n_estimators=100, min_samples_leaf=10)
+        ContinuousModel = RandomForestRegressor(n_estimators=100, min_samples_leaf=10)
+    else:
+        BinaryModel = LogisticRegressionCV(
```

Comment:
if you want to implement logistic regression with lasso penalty as was done in the R implementation, not the right parameters (you need to set penalty='l1', and the solver accordingly)